src/queue.cpp

```diff
@@ -101,6 +101,10 @@ cl_int cvk_command_queue::satisfy_data_dependencies(cvk_command* cmd) {
             continue;
         }
         CVK_ASSERT(mem->is_image_type());
+        if (config.init_image_at_creation()) {
+            tracker.set_state(cvk_mem_init_state::completed);
```

Comment:
Shouldn't this be done in `cvc_image::init_vulkan_image`, after completing the execution of the init command?

Reply:
Yes, that's better, changed made